bundles/AdminBundle/Resources/public/js/pimcore/helpers.js

```diff
@@ -1790,17 +1790,9 @@ pimcore.helpers.editmode.openLinkEditPanel = function (data, callback) {
         listeners: {
             keyup: function (el) {
                 const value = el.getValue();
+                const pathRegex = new RegExp('^((/)|((/[^/]+)+(/)?))$');
```

Comment:
Are those groups really necessary? I don't thing we need them, because we don't use them.

```suggestion
                const pathRegex = new RegExp('^(/|(/[^/]+)+/?)$');
```

Reply:
@jdreesen sorry, didn't see the second comment yet. Had an issue without the groups on https://regex101.com/ while testing. Couldn't figured out what's wrong, so I left it as is. Updated as per your suggestion :-)